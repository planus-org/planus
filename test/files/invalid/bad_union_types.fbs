union A0 {
  A: A0,
  B: A1,
  C: [bool],
  D: [bool: 1],
  E: bool,
  F: uint8,
  G: float32,
  H: [A0],
  I: [AI],
  A0,
  A1,
  [bool],
  [bool: 1],
  bool,
  uint8,
  float32,
  string,
  [A0],
  [A1],
}

enum A1: uint8 {}
