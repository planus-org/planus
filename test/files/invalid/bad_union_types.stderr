error: Unsupported type
  ┌─ test/files/invalid/bad_union_types.fbs:2:6
  │
2 │   A: A0,
  │      ^^ Unions in unions are not supported

error: Unsupported type
  ┌─ test/files/invalid/bad_union_types.fbs:3:6
  │
3 │   B: A1,
  │      ^^ Enums in unions are not supported

error: Unsupported type
  ┌─ test/files/invalid/bad_union_types.fbs:4:6
  │
4 │   C: [bool],
  │      ^^^^^^ Vectors in unions are not supported

error: Unsupported type
  ┌─ test/files/invalid/bad_union_types.fbs:5:6
  │
5 │   D: [bool: 1],
  │      ^^^^^^^^^ Arrays in unions are not supported

error: Unsupported type
  ┌─ test/files/invalid/bad_union_types.fbs:6:6
  │
6 │   E: bool,
  │      ^^^^ Bools in unions are not supported

error: Unsupported type
  ┌─ test/files/invalid/bad_union_types.fbs:7:6
  │
7 │   F: uint8,
  │      ^^^^^ Integers in unions are not supported

error: Unsupported type
  ┌─ test/files/invalid/bad_union_types.fbs:8:6
  │
8 │   G: float32,
  │      ^^^^^^^ Floats in unions are not supported

error: Unsupported type
  ┌─ test/files/invalid/bad_union_types.fbs:9:6
  │
9 │   H: [A0],
  │      ^^^^ Vectors in unions are not supported

error: Unknown type
   ┌─ test/files/invalid/bad_union_types.fbs:10:7
   │
10 │   I: [AI],
   │       ^^

error: Unsupported type
   ┌─ test/files/invalid/bad_union_types.fbs:11:3
   │
11 │   A0,
   │   ^^ Unions in unions are not supported

error: Unsupported type
   ┌─ test/files/invalid/bad_union_types.fbs:12:3
   │
12 │   A1,
   │   ^^ Enums in unions are not supported

error: Unsupported type
   ┌─ test/files/invalid/bad_union_types.fbs:13:3
   │
13 │   [bool],
   │   ^^^^^^ Vectors in unions are not supported

error: Unsupported type
   ┌─ test/files/invalid/bad_union_types.fbs:14:3
   │
14 │   [bool: 1],
   │   ^^^^^^^^^ Arrays in unions are not supported

error: Unsupported type
   ┌─ test/files/invalid/bad_union_types.fbs:15:3
   │
15 │   bool,
   │   ^^^^ Bools in unions are not supported

error: Unsupported type
   ┌─ test/files/invalid/bad_union_types.fbs:16:3
   │
16 │   uint8,
   │   ^^^^^ Integers in unions are not supported

error: Unsupported type
   ┌─ test/files/invalid/bad_union_types.fbs:17:3
   │
17 │   float32,
   │   ^^^^^^^ Floats in unions are not supported

error: Unsupported type
   ┌─ test/files/invalid/bad_union_types.fbs:18:3
   │
18 │   string,
   │   ^^^^^^ Strings in unions without variant names are not supported

error: Unsupported type
   ┌─ test/files/invalid/bad_union_types.fbs:19:3
   │
19 │   [A0],
   │   ^^^^ Vectors in unions are not supported

error: Unsupported type
   ┌─ test/files/invalid/bad_union_types.fbs:20:3
   │
20 │   [A1],
   │   ^^^^ Vectors in unions are not supported

